proto/v1/database_service.proto

```diff
@@ -137,6 +152,24 @@ message GetDatabaseSchemaRequest {
   string name = 1 [(google.api.field_behavior) = REQUIRED];
 }
 
+message GetBackupSettingRequest {
+  // The name of the database to retrieve backup setting.
+  // Format: environments/{environment}/instances/{instance}/databases/{database}/backupSetting
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message UpdateBackupSettingRequest {
+  // The name of the database to update the backup setting.
+  // Format: environments/{environment}/instances/{instance}/databases/{database}/backupSetting
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Also, we don't need this name, it's part of BackupSetting below.

Reply:
Done.